src/runtime/node/tls/server.ts

```diff
@@ -0,0 +1,10 @@
+import type tls from "node:tls";
+import { notImplemented } from "../../_internal/utils";
+import { Server as _Server } from "node:net";
+
+export class Server extends _Server implements tls.Server {
+  addContext(hostname: string, context: tls.SecureContextOptions) {}
+  setSecureContext(options: tls.SecureContextOptions) {}
+  setTicketKeys(keys: Buffer) {}
+  getTicketKeys = notImplemented("Server.getTicketKeys");
```

Comment:
we should avoid class prop assignment syntax. also, I guess we could simply couple `set/get` using an internal array?

Reply:
Unfortunately it seems a little more complicated than that. `ticketKeys` is actually a Buffer, and I'm not super sure of the semantics here. FWIW even [bun's implementation](https://github.com/oven-sh/bun/blob/4b8f89cb73ac2b38f1da6a28eb91861eb7222264/src/js/node/tls.ts#L627-L633) throws not implemented.

Fixed the prop assignment syntax though.